File: RevisaoImcAtleta/RevisaoImcAtleta/ViewController.m
//
//  ViewController.m
//  RevisaoImcAtleta
//
//  Created by Usuário Convidado on 24/03/17.
//  Copyright © 2017 Joyce. All rights reserved.
//

#import "ViewController.h"
#import "Pessoa.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)calcular:(id)sender {
    Pessoa *p = [[Pessoa alloc]init];
    
    p.peso = self.txtPeso.text.floatValue;
    p.altura = self.txtAltura.text.floatValue;
    p.nome = self.txtNome.text;
    
    [p calcularIMC];
    self.txtIMC.text= [NSString stringWithFormat:@"%f",p.imc];
    
}
@end

File: Navegando Entre Telas/Navegando Entre Telas/ViewController.swift
//
//  ViewController.swift
//  Navegando Entre Telas
//
//  Created by Usuário Convidado on 12/05/17.
//  Copyright © 2017 Joyce Silva. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func fechar(_ sender: Any) {
        self.dismiss(animated: true, completion: nil)
    }

}

